src/EFCore.PG/Query/Expressions/Internal/PgFunctionExpression.cs

```diff
@@ -217,25 +250,13 @@ protected override Expression Accept(ExpressionVisitor visitor)
                 : base.Accept(visitor);
         }
 
-        /// <summary>
-        ///     Reduces the node and then calls the <see cref="ExpressionVisitor.Visit(Expression)" /> method passing the
-        ///     reduced expression.
-        ///     Throws an exception if the node isn't reducible.
-        /// </summary>
-        /// <param name="visitor"> An instance of <see cref="ExpressionVisitor" />. </param>
-        /// <returns> The expression being visited, or an expression which should replace it in the tree. </returns>
-        /// <remarks>
-        ///     Override this method to provide logic to walk the node's children.
-        ///     A typical implementation will call visitor.Visit on each of its
-        ///     children, and if any of them change, should return a new copy of
-        ///     itself with the modified children.
-        /// </remarks>
+        /// <inheritdoc />
         protected override Expression VisitChildren(ExpressionVisitor visitor)
         {
-            var newInstance = Instance != null ? visitor.Visit(Instance) : null;
+            var newInstance = visitor.Visit(Instance);
```

Comment:
I'm not too sure about this, aren't we supposed to check for null before calling `visitor.Visit()`, like before? What's the pattern you're seeing in other call sites?

Reply:
OK, sounds good - let's keep your change then.